src/duplicacy_snapshotmanager.go

```diff
@@ -896,10 +898,25 @@ func (manager *SnapshotManager) CheckSnapshots(snapshotID string, revisionsToChe
 
 
     if showStatistics {
-        for snapshotID, snapshotList := range snapshotMap {
+        tableBuffer := new(bytes.Buffer)
```

Comment:
This new format definitely looks much nicer, but I wonder if you can add another switch to enable this format, as some users may have already been parsing the output in their scripts.

Something like `duplicacy check -stats` -> old format, and `duplicacy check -stats -tabular` -> new format.

Reply:
Sounds good.